packages/augur-test/libs/LocalAugur.ts

```diff
@@ -37,7 +37,14 @@ function makeDeployerConfiguration() {
   return new DeployerConfiguration(contractInputRoot, artifactOutputRoot, augurAddress, createGenesisUniverse, isProduction, useNormalTime, legacyRepAddress);
 }
 
-export async function makeTestAugur(accounts: AccountList): Promise<Augur<any>> {
+interface UsefulContractObjects {
+  provider: EthersProvider;
+  signer: EthersFastSubmitWallet;
+  dependencies: ContractDependenciesEthers;
+  compiledContracts: CompilerOutput;
+  addresses: any;
```

Comment:
Shouldn't this be changed to match the type returned by `ContractDeployer.deploy`?

Reply:
Yeah it should.